spacelift/data_aws_integrations_test.go

```diff
@@ -0,0 +1,37 @@
+package spacelift
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	. "github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/testhelpers"
+)
+
+func TestAWSIntegrationsData(t *testing.T) {
+	randomID := acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum)
+	resourceName := "spacelift_aws_integration.test"
+	datasourceName := "data.spacelift_aws_integrations.test"
+
+	testSteps(t, []resource.TestStep{{
+		Config: fmt.Sprintf(`
+		 resource "spacelift_aws_integration" "test" {
```

Comment:
I think it's worth creating multiple integrations in the test. That way we'll have confidence the code that maps the result to the TF objects isn't doing anything weird.

Reply:
I did an overhaul of the tests. Now the test checks all the fields and it also creates two resources (and checks both of them)